app/screens/home/maximize_screen.tsx

```diff
@@ -114,10 +118,7 @@ export default function MaximizeScreen({
                 {postUser?.name}
               </ThemedText>
               <ThemedText colorType="white" type="small">
-                {"on " +
-                  postDate.toLocaleDateString() +
-                  ", at " +
-                  postDate.toLocaleTimeString()}
+                {"on " + postDate.toUTCString()}
```

Comment:
Why not render the dateTime in date then time as previously ? It's clearer

Reply:
Because it didn't work on testing and this way it works using the Timestamp